src/index.html

```diff
@@ -1,14 +1,85 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#page-top" class="logo">
+        <img
+          class="logo__img"
+          src="./images/logo.png"
+          alt="Moyo header logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="hover">
```

Comment:
```suggestion
            <a
              href="#laptops"
              class="nav__link"
              data-qa="hover"
            >
```

check formatting please in all places